vector/src/main/java/im/vector/app/features/spaces/SpaceListViewModel.kt

```diff
@@ -134,6 +130,11 @@ class SpaceListViewModel @AssistedInject constructor(
                 .launchIn(viewModelScope)
     }
 
+    private fun roomsInSpaceFilter() = when {
```

Comment:
Please do not change it now, but the `fun` name is a bit strange to me.
Maybe something like `getSpaceFilter()` would be better.
On the fund, I do not really get why we have such filter here, since IIUC this ViewModel is used by the screen which displays only Spaces. So having orphans rooms here is strange to me.

Reply:
Ah, OK, it's to update the notification count. Please ignore me!